test/OpenTelemetry.Instrumentation.Hangfire.Tests/OpenTelemetry.Instrumentation.Hangfire.Tests.csproj

```diff
@@ -9,6 +9,7 @@
   <ItemGroup>
     <PackageReference Include="Hangfire.Core" Version="1.8.1" />
     <PackageReference Include="Hangfire.MemoryStorage" Version="1.7.0" />
+    <PackageReference Include="Moq" Version="$(MoqPkgVer)" />
```

Comment:
We don't want to take in dependency on Moq. Check https://github.com/open-telemetry/opentelemetry-dotnet/issues/4758

You should be able to add a custom test processor implementation instead.